content/blog/2023-04-05-response-to-forbes-article-about-ethereum-classic/index.md

```diff
@@ -0,0 +1,340 @@
+
+With regards to “resistance to change”, ETC has no resistance to change when it is about upgrading its technology, adding new useful features, fixing bugs, or keeping up with the EVM standard as described previously.
+
+Therefore, the contradiction in which the Forbes argument above incurs is that if the Ethereum community is working to improve the scaling situation, thanks to the much greater attention on it from developers, whatever improvements they may create will be implemented in Ethereum Classic as well.
+
+Again, ETH and ETC, except for the consensus layer, are practically the same technology stack. In fact, ETC core developers are active participants in the larger EVM format ecosystem, constantly working to integrate new EIPs into ETC or participating in the debates and definitions of these protocols.
+
+**For more information:**
+
+- The Format War, Layering, and Systemic Risk Will Define the Future Landscape of the Blockchain Industry: https://etherplan.com/2019/06/15/the-format-war-layering-and-systemic-risk-will-define-the-future-landscape-of-the-blockchain-industry/7820/ 
+
+- The EVM Object Format (EOF) Upgrade Explained: https://ethereumclassic.org/blog/2023-01-17-the-evm-object-format-eof-upgrade-explained 
+
+## Section: Ethereum Classic Price
+
+### The first paragraph of this section reads:
+
+*“Ethereum has moved from strength to strength over the years, becoming the second largest crypto by market capitalization after Bitcoin.”*
+
+### Our response:
+
+To the contrary, Ethereum has moved from strength to weakness and Ethereum Classic has moved from weakness to strength.
+
+Weakness in Ethereum is shown in two dimensions. The first is that it discarded proof of work as the consensus mechanism eliminating the major improvements that this technology brought to the world, namely to reach consensus with a 51% fault tolerance on a global scale, on a peer-to-peer, fully replicated network, with the highest levels of permissionlessness and censorship resistance man has ever seen.
+
+The second one is that to build a proof of stake system, because of its inherent insecurity, developers have to create an incredibly complex protocol, with incredibly complex software clients (indeed, now to run an Ethereum proof of stake node, operators need to run two clients at the same time, not one) with ten times or more lines of code than in proof of work systems.
+
+This complexity and volume in itself is a security hole because, by now, no one in the world may understand the whole Ethereum system and such amounts of information and moving parts only create hazards and risks where hackers and attackers may put their hands on to corrupt and tamper the system.
```

Comment:
Line 260:
```suggestion
This complexity and volume in itself is a security hole because, by now, no one in the world may understand the whole Ethereum system and such amounts of information and moving parts only create hazards and risks where hackers and attackers may put their hands to corrupt and tamper with the system.
```